package/boot/uboot-mediatek/Makefile

```diff
@@ -776,6 +776,10 @@ UBOOT_TARGETS := \
 	mt7988_rfb-emmc \
 	mt7988_rfb-sd
 
+UBOOT_CUSTOMIZE_CONFIG := \
+	--disable TOOLS_KWBIMAGE \
+	--disable TOOLS_LIBCRYPTO
```

Comment:
I am not confident in this change. It seems that the original patch `no-kwbimage.patch
` was intended to avoid using OpenSSL.

Reply:
I had all this ready in my local tree already and tested on most MTK chips and devices I got here. I also removed `no-kwbimage.patch` in favor of using the now available config options.
Another thing which was annoying was the bootmenu now adds some UEFI boot entries which won't work, and one had to add `# CONFIG_CMD_BOOTEFI_BOOTMGR is not set` for all boards for it to be as before.
Please include that change as well, so existing bootmenus won't be polluted with (anyway non-working) UEFI stuff.

Reply:
Thanks! Fixed.

BTW, the newly added `# CONFIG_CMD_BOOTEFI_BOOTMGR is not set` will be automatically removed for some devices in the second refresh commit because they don't have some `EFI` dependencies.
```
mt7622_ubnt_unifi-6-lr-v1_defconfig
mt7622_ubnt_unifi-6-lr-v2_defconfig
mt7622_ubnt_unifi-6-lr-v3_defconfig
ravpower-rp-wd009-ram_defconfig
mt7986_xiaomi_redmi-ax6000_defconfig
mt7981_xiaomi_mi-router-ax3000t_defconfig
mt7981_nokia_ea0326gmp_defconfig
mt7622_xiaomi_redmi-router-ax6s-ubi-loader_defconfig
```

Reply:
Yes, makes sense, only device which got `CONFIG_CMD_BOOTEFI` enabled should require the line to be added. And having `bootefi` command is completely optional and not required for OpenWrt. It can still be useful, e.g. to run Memtest86+, so on devices with enough space it makes sense to keep it.
However, `CONFIG_CMD_BOOTEFI_BOOTMGR` doesn't make sense unless we'd want to boot via UEFI by default (and that would imho be a bad idea, as UEFI doesn't provide any meaningful way to implement dual-boot while avoiding any single point of failure).